jenkins-pipelines/performance/scylla-enterprise/scylla-enterprise-perf-regression-latency-650gb-grow-shrink-tablets.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    availability_zone: 'a,b,c',
```

Comment:
I just spotted we set this test to be multi-az, while jenkins pipeline resets it anyway to 1 zone.
So if this test meant to be multi az, it is not triggered this way.

Reply:
it was meant to be multi-az, jenkins pipeline should be using what's written here as default
or that our trigger is limiting it...

Reply:
Do they want it to run in multi-AZ?
Is our job that runs in branch-perf-v14 runs in multi-AZ as well?

Reply:
Why do we have this job duplicated for master and for enterprise?
Perf jobs shouldn't be duplicated so we can compare releases in one email.

Reply:
again part of the refactoring, I want each directory to match the directory we are using 
at the beginning I've used symlinks, but jenkins is stupid and can't work with symlink correctly

Reply:
regardless I've disable the automatic creation for the perf jobs,
there is an inherent problem with building from master for the v14 branch (location of the jobs in the source is different)

so we will be generating from master, only once we'll be running from master
BTW @avikivity was asking about it last week, in regards of tablet related changes that would be done by the team working on tablet would be done on master, and we won't be backporting it to v14. 

I think we should start doing perf runs on master.

Reply:
We won’t do perf runs using master because it will make the tests even more flakey.

what about the multi AZ?
Is it needed here?
Is this how we run it in the original job?

Reply:
> Do they want it to run in multi-AZ? Is our job that runs in branch-perf-v14 runs in multi-AZ as well?

They already do, I don't know if they cared about it or not.
and yes it's multi-AZ also on v14

Reply:
> We won’t do perf runs using master because it will make the tests even more flakey.

this whole process of creating a new branch once a year, and vetting it, isn't any better.
and the end result we have flakiness we can't detect for months regardless.
working on a branch doesn't make this process any easier.

> 
> what about the multi AZ? Is it needed here? Is this how we run it in the original job?

Reply:
It is.
You won't convince me otherwise.
We don't have to use the same branch, each new test that is not backporting easily can run from a new branch.

Reply:
that mean keep manually setup of the job, and having not real visibility or control on what actually runs, as those expend to more and more cases.

it's not not gonna be maintainable like that, and call you took 6 years ago, when SCT was in a condition master could be run at all. it's not the same situation today.

I won't be maintaining a branch per test, and you won't convince otherwise...